packages/env/README.md

```diff
@@ -668,6 +680,26 @@ You can tell `wp-env` to use a specific PHP version for compatibility and testin
 }
 ```
 
+#### Post-Install Command
```

Comment:
I think it'd be nice to document pointing to your own script! (node or shell)

Reply:
This _might_ be a little superfluous since you can run arbitrary commands, but, more examples can't hurt!

Reply:
True! I feel like it could be easy to miss if you aren't super familiar with it